src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components';
+
+export interface CheckboxProps {
+	label?: string | React.ReactNode;
+	onChange: (checked: boolean) => void;
+	checked: boolean;
+}
+
+export const Checkbox: React.FC<CheckboxProps> = ({ label, onChange, checked, ...props }) => {
```

Comment:
nice components. We can port that over to the ui lib. Pretty sure we are gonna use that in the staking app and in the v3gm interface as well. 
https://github.com/synthetixio/ui
Can you make a TODO here and create an issue in this repo here so we don't forget

Reply:
created the issue:
https://github.com/Synthetixio/ui/issues/28